.github/workflows/release.yml

```diff
@@ -17,15 +17,17 @@ jobs:
         with:
           node-version: 'lts/*'
 
-      - name: Configure SENTRY env vars
-        run:
-          node ./build/configure-sentry.js >> $GITHUB_ENV
-
       - name: Install Sentry CLI
         run: |
           curl -sL https://sentry.io/get-cli/ | bash
           sentry-cli --version
 
+      - name: Configure SENTRY env vars
```

Comment:
Let's install the Sentry CLI before setting up env vars, so that we don't accidentally leak our env vars to the Sentry installer script.